app/models/shared_with.rb

```diff
@@ -1,8 +1,9 @@
 class SharedWith
   NAME_VALUES = HashWithIndifferentAccess.new(
-    team:     'Just the team',
-    internal: 'Other teams internally',
-    external:  'Other teams externally'
+    anonymised: I18n.t('helpers.label.storing_information.shared_with.anonymised'),
+    team:       I18n.t('helpers.label.storing_information.shared_with.team'),
+    internal:   I18n.t('helpers.label.storing_information.shared_with.internal'),
+    external:   I18n.t('helpers.label.storing_information.shared_with.xternal')
```

Comment:
Slight typo here: `xternal`

Reply:
(practical upshot: you'll see a "translation missing" message in the HTML source of the output and ActiveSupport will try to infer something reasonable)